.github/workflows/dev-infrastructure.yml

```diff
@@ -27,10 +27,11 @@ jobs:
       - name: 'az bicep format and lint'
         working-directory: './dev-infrastructure'
         run: |
+          az bicep version
```

Comment:
did you want to keep this?

What do you think about an `az bicep upgrade` here as well?

Reply:
I did, just so ease troubleshooting - yesterday I had az bicep upgrade and it didn't actually upgrade in the GitHub Actions runner.

Take a look at this attempt from yesterday https://github.com/Azure/ARO-HCP/actions/runs/9006416096/job/24743908959 in the `az bicep format and lint` step

```
Run az bicep upgrade
WARNING: The configuration value of bicep.use_binary_from_path has been set to 'false'.
Installing Bicep CLI v0.2[6](https://github.com/Azure/ARO-HCP/actions/runs/9006416096/job/24743908959#step:3:7).170...
Successfully installed Bicep CLI to "/home/runner/.azure/bin/bicep".
set -e; \
FILES="$(find . -type f -name "*.bicep*")"; \
for file in $FILES; do \
echo "az bicep format --file ${file}"; \
az bicep format --file $file; \
done
az bicep format --file ./modules/aks-cluster-base.bicep
WARNING: A new Bicep release is available: v0.2[7](https://github.com/Azure/ARO-HCP/actions/runs/9006416096/job/24743908959#step:3:8).1. Upgrade now by running "az bicep upgrade".
```